addons/mail/controllers/thread.py

```diff
@@ -102,6 +102,7 @@ def mail_message_update_content(self, message_id, body, attachment_ids, attachme
             raise NotFound()
         if not message_sudo.model or not message_sudo.res_id:
             raise NotFound()
+        body = Markup(body) if body else body  # may contain HTML such as @mentions
```

Comment:
Seems like we markup before escaping it in the method, shouldn't it be reversed? Might be missing something